docs: adapt docs to new story (#5800)
jina-ai/jina: docs/concepts/preliminaries/coding-in-python-yaml.md

```{glossary}

**Pythonic**

>     The Flow, Deployment and Executors are all written in Python files, and the entrypoint is via Python.

Reviewer: ```suggestion
    Flows, Deployments and Executors are all written in Python files, and the entrypoint is via Python.
```

```
````

> In general, the YAML style can be used to represent and configure the objects Flow and Deployment which are the objects orchestrating the serving of Executors and applications. 

Reviewer: ```suggestion
In general, the YAML style can be used to represent and configure a Flow or Deployment which are the objects orchestrating the serving of Executors and applications.
```

> The YAMLish style separates the Flow or Deployment representation from the logic code from Executors. It is more flexible to configure and should be used for more complex projects in production. In many integrations such as JCloud, Kubernetes, YAMLish is preferred. 

Reviewer: ```suggestion
The YAMLish style separates the Flow or Deployment representation from the logic code from Executors. It is more flexible to configure and should be used for more complex projects in production. In many integrations such as JCloud and Kubernetes, YAMLish is preferred. 
```
Why YAMLish and not YAML? Are we discussing same YAMLness as above?
